test/TraceListener.netcoreapp10.Tests/TraceListener.netcoreapp10.Tests.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.ApplicationInsights.TraceListener.Tests</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.TraceListener.Tests</AssemblyName>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\TraceListener.Net45.Tests\ApplicationInsightsTraceFilterTests.cs" Link="ApplicationInsightsTraceFilterTests.cs" />
```

Comment:
moving thos to the folder with the name Shared can make folders structure cleaner. Can you do it please?

Reply:
Done